src/components/DuffelCardForm/lib/types.ts

```diff
@@ -55,45 +82,87 @@ export interface DuffelCardFormProps {
    */
   tokenProxyEnvironment?: "development" | "staging" | "production";
 
+  /**
+   * The card intent defines what the form is meant to look like.
+   * It can be one of:
+   *
+   * - `to-create-card-for-temporary-use`: The full form will be shown. You may also use this intent for the use case of using and saving the card.
+   * - `to-use-saved-card`: When using this intent also provide the saved card ID. Only a cvv field will be rendered.
+   * - `to-save-card`: The form will be shown without the cvv field.
```

Comment:
```suggestion
   * - `to-save-card`: The form will be shown without the cvv field. This allows you to save a card for future use, but not use it immediately.
```
Or something like that?

Reply:
Good shout, I expanded that a bit more to also mention the save during checkout use case.